modules/proc_use-api.adoc

```diff
@@ -104,6 +101,77 @@ Use the following procedure to create an OAuth access token.
 This is the only opportunity to copy and save the access token. It cannot be reobtained after leaving this page. 
 ====
 
+[id="reassigning-oauth-access-token"]
+== Reassigning an OAuth access token
+
+{productname} superusers of a registry can take control of API tokens created by other users and re-assign those tokens to a different user. This helps audit logs properly reflect which member of an organization used, or is using, a token after reassignment.
```

Comment:
"superusers of a registry"  can be change to  like "organization administrator", in this release superusers doesn't have any special privilege